dataactvalidator/filestreaming/fieldCleaner.py

```diff
@@ -120,6 +122,32 @@ def cleanLength(length):
             raise ValueError("Length must be positive")
         return length
 
+    @staticmethod
+    def cleanRow(row):
+        for key in row.keys():
+            value = row[key]
+            if value is not None:
```

Comment:
Do we want to consider what the type of the field is supposed to be, we should be able to get that out of file_columns.field_type_id.  Possibly the answer is no, and if someone puts "9,4" into a random string we want to convert it to "94".  I just want to raise the point that as written this will apply to all strings that can be converted to ints.